ldi-core/ldes-client/tree-node-supplier/src/main/java/ldes/client/treenodesupplier/repository/sqlite/EntityManagerFactory.java

```diff
@@ -37,6 +38,7 @@ public void destroyState() {
 		try {
 			em.close();
 			emf.close();
+			instance = null;
```

Comment:
can 't you achieve the same with `instance.destroyState();` ?

Reply:
In this case, that would call itself recursively.